### DISCUSSION TITLE
External links - can internal button/link blade files be reused and how?

### DISCUSSION THREAD

**Discussion by hrvoj3e** [filamentphp/filament#4501] [Help]
I am trying to render a button or plain link with external url on ViewRecord page.
 - E.g. `vendor/filament/support/resources/views/components/button.blade.php`

Can someone give one simple example how to implement a component which uses `Filament\Support\Actions\Concerns\CanOpenUrl`?

I see that `vendor/filament/forms/resources/views/components/actions/action.blade.php` uses 
```
<x-dynamic-component
    :href="$action->isEnabled() ? $action->getUrl() : null"
>
    {{ $slot }}
</x-dynamic-component>
```

I tried something like this. Used placeholder as a start value (it has label, hint, ...) but cannot figure out how to proceed...
```
<?php

namespace App\Filament\Forms\Components;

use Filament\Forms\Components\Placeholder;
use Filament\Support\Actions\Concerns;

class Button extends Placeholder
{
    use Concerns\CanOpenUrl;

    public $slot = 'slot-content'; // hack just for test to make it work

    protected string $view = 'forms::components.button';
}
```

**Comment by danharrin** [comment]
`<x-forms::button tag="a" href="https://...">`, `<x-filament::button tag="a" href="https://...">`
